scripts/graphics/tilesview.js

```diff
@@ -40,6 +44,7 @@ define([
                             );
                         }
                     }
+                    return null;
```

Comment:
This function is supposed to return something. It worked before because functions return 'undefined' by default, but I prefer being explicit.